code/src/pages/MovieList.js

```diff
@@ -0,0 +1,22 @@
+import React, {useState, useEffect} from 'react'
+import { MovieCard } from '../components/MovieCard'
+import { TOP_RATED_URL } from '../reusable/urls'
+export const MovieList = () => {
+  
+  const [movies, setMovies] = useState([]);
```

Comment:
There should be an empty line between 1 and 2 and between 3 and 4, useState should be on the first line inside the component.